### PR TITLE
SONAR-6680 Move computation of measure Quality Profile to Compute Engine

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: sonar-batch/src/main/java/org/sonar/batch/rule/QProfileSensor.java [hunk 1]
```diff
@@ -29,6 +29,8 @@
 
 /**
  * Stores which Quality profiles have been used on the current module.
+ *
+ * TODO This information should not be stored as a measure but should be send as metadata in the {@link org.sonar.batch.protocol.output.BatchReport}
```
Comment (SonarTech) [inline sonar-batch/src/main/java/org/sonar/batch/rule/QProfileSensor.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

File: server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeQProfileMeasureStep.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import org.sonar.server.computation.qualityprofile.QPMeasureData;
+import org.sonar.server.computation.qualityprofile.QualityProfile;
+
+import static org.sonar.server.computation.component.Component.Type.MODULE;
+import static org.sonar.server.computation.component.ComponentVisitor.Order.POST_ORDER;
+
+/**
+ * Aggregate which Quality profiles have been used each last module
+ *
+ * TODO should we really persist measures on module levels ?
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeQProfileMeasureStep.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* INFO 2 info

Watch the comments in this conversation to review them.

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* INFO 2 info

Watch the comments in this conversation to review them.

**Comment by sns-seb** [comment]
well, a few question which do not imply code change. LGTM

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.

**Pull request closed without merging**